### DISCUSSION TITLE
Please publish a github release for JitPack to access

### DISCUSSION THREAD

**Discussion by GedMarc** [Auties00/WhatsappWeb4j#32] [Ideas]
Jitpack allows developers to reference builds directly from github by tag or branch, 
Currently though it looks like the auto build does not support JDK 16

This results in the below log when using JitPack 
https://jitpack.io/com/github/Auties00/WhatsappWeb4j/master-whatsappweb4j-1.2-gd6edb3d-77/build.log

https://jitpack.io/#Auties00/WhatsappWeb4j/master-SNAPSHOT
Not sure if this would be something you are interested in supporting really 

[![](https://jitpack.io/v/Auties00/WhatsappWeb4j.svg)](https://jitpack.io/#Auties00/WhatsappWeb4j)

**Comment by Auties00** [comment]
> Jitpack allows developers to reference builds directly from github by tag or branch,
> Currently though it looks like the auto build does not support JDK 16
> 
> This results in the below log when using JitPack
> https://jitpack.io/com/github/Auties00/WhatsappWeb4j/master-whatsappweb4j-1.2-gd6edb3d-77/build.log
> 
> https://jitpack.io/#Auties00/WhatsappWeb4j/master-SNAPSHOT
> Not sure if this would be something you are interested in supporting really
> 
> [![](https://camo.githubusercontent.com/91fafa2ee6ccaf7a1df403cb8a73a43135ae5c81c94b12f86d1f417851419f2d/68747470733a2f2f6a69747061636b2e696f2f762f41757469657330302f5768617473617070576562346a2e737667)](https://jitpack.io/#Auties00/WhatsappWeb4j)

Looks interesting, I'll add it before version 2.0 comes out. There's a problem though as the unit tests for the build require a WhatsApp connection to exist, which is not really something that can be automated. I've not managed to come up with a fix for this for Github's Maven CI, so I guess I'll do the same for jitpack. Le me know if you have better ideas
